extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -86,6 +88,8 @@ require([
 			sourcePoint.initDetection();
 		}
 
+		pageFair.initDetection();
```

Comment:
Don't we want to init it only if the flag is enabled (`context.opts.pageFairDetection`)?